### PR TITLE
pixpin1.9.5.0 (new cask)

### PR DESCRIPTION
**Pull request by WarmEpoch**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/p/pixpin.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  sha256 "30e43609473a4894d880c8c0774f43c1d91497df9639c0e72ceef7d9ca275970"
+
+  url "https://download.pixpinapp.com/PixPin_#{version}.dmg"
+  name "PixPin"
+  name "截图"
+  name "贴图"
+  name "长截图"
+  name "文字识别"
+  name "标注"
+  desc "功能强大使用简单的截图/贴图工具，帮助你提高效率"
```
Comment (bevanjkay) [inline Casks/p/pixpin.rb:12]: Please provide the `desc` in English.

File: Casks/p/pixpin.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  sha256 "30e43609473a4894d880c8c0774f43c1d91497df9639c0e72ceef7d9ca275970"
+
+  url "https://download.pixpinapp.com/PixPin_#{version}.dmg"
+  name "PixPin"
+  name "截图"
+  name "贴图"
+  name "长截图"
+  name "文字识别"
+  name "标注"
+  desc "Powerful and easy-to-use screenshot/sticker tool to help you be more productive"
```
Comment (bevanjkay) [inline Casks/p/pixpin.rb:12]:
```suggestion
  desc "Screenshot tool"

File: Casks/p/pixpin.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  version "1.9.5.0"
+  sha256 "30e43609473a4894d880c8c0774f43c1d91497df9639c0e72ceef7d9ca275970"
+
+  url "https://download.pixpinapp.com/PixPin_#{version}.dmg"
+  name "PixPin"
+  name "截图"
+  name "贴图"
+  name "长截图"
+  name "文字识别"
+  name "标注"
```
Comment (bevanjkay) [inline Casks/p/pixpin.rb:11]:
```suggestion
```

These seem like descriptors, not the application name.

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @WarmEpoch

File: Casks/p/pixpin.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  sha256 "30e43609473a4894d880c8c0774f43c1d91497df9639c0e72ceef7d9ca275970"
+
+  url "https://download.pixpinapp.com/PixPin_#{version}.dmg"
+  name "PixPin"
+  desc "Screenshot tool"
+  homepage "https://pixpinapp.com/"
+
+  livecheck do
+    url "https://pixpinapp.com/"
+    regex(/href=.*PixPin[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
```
Comment (khipp) [inline Casks/p/pixpin.rb:12]:
```suggestion
    url :homepage
    regex(/href=.*?PixPin[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
```

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**